### PR TITLE
Rename retained_versions field for consistency

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #9030

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: docs/workflows/repo-versioning.rst [hunk 1]
```diff
@@ -9,21 +9,20 @@ Version Retention
 -----------------
 
 .. warning::
-    Support for retained_versions is provided as a tech preview in Pulp 3.  Functionality may not
+    Support for retain_repo_verisons is provided as a tech preview in Pulp 3.  Functionality may not
     work or may be incomplete. Also, backwards compatibility when upgrading is not guaranteed.
```
Comment (daviddavis) [inline docs/workflows/repo-versioning.rst:13]: This feature is in tech preview which I think allows us to rename the field.

**Review by ipanova — APPROVED** [review]

**Comment by dralley** [comment]
As per the pulp-meeting discussion, we should avoid merging any PRs containing migrations before Friday to enable the possibility for 3.14.z-stream migrations.  And we'll decide then how to proceed.

File: docs/workflows/repo-versioning.rst [hunk 2]
```diff
@@ -9,21 +9,20 @@ Version Retention
...
-the latest version.
+Setting retain_repo_versions to 1 effectively disables repository versioning since Pulp will only
+store the latest version.
 
 To update this field for a file Repository called myrepo, simply call:
 
-```
-pulp file repository update --name myrepo --retained-versions 1
-```
+``` pulp file repository update --name myrepo --retained-versions 1 ```
```
[the remark this answers is not in the document]
Comment (daviddavis) [reply docs/workflows/repo-versioning.rst:25]: Here we go: https://github.com/pulp/pulp-cli/issues/298

Thanks @ggainey

File: manage.py [hunk 3]
```diff
@@ -1 +0,0 @@
-pulpcore/app/manage.py
```
Comment (dralley) [inline manage.py:1]: Was this change intentional?

**Review by dralley — APPROVED** [review]
